gym_xiangqi/xiangqi_game.py

```diff
@@ -35,6 +35,9 @@ def __init__(self):
         self.counter = COUNT
         self.agent_kills = []
         self.enemy_kills = []
+        self.cur_selected_pid = 0
```

Comment:
Oh this should be fine we have `self.cur_selected` and `self.cur_selected_pid`. I added `self.cur_selected_pid` because we needed a way to particularize the piece the player moved. Lines like `if self.cur_selected is None:` uses  `self.cur_selected` which is initialized as `None`.